packages/eslint-plugin-angular-template-spacing/src/schematics.ts

```diff
@@ -0,0 +1,46 @@
+import * as chalk from 'chalk';
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import {
+  isJsonArray,
+  isJsonObject, JsonObject,
+} from '@angular-devkit/core';
+
+function createOrPushToArray(json: JsonObject, path: string, member: string) {
+  const array = json[path];
+  if (isJsonArray(array)) {
+    if (!array.includes(member)) {
+      array.push(member);
+    }
+  } else {
+    json[path] = [member];
+  }
+}
+
+// You don't have to export the function as default. You can also have more than one rule factory
+// per file.
+export function ngAdd(_options: any): Rule {
+  return (tree: Tree, _context: SchematicContext) => {
+    const path = '.eslint.json';
+    const pluginName = '@betsys-eslint/eslint-plugin-angular-template-spacing'
+    const pluginSettings = `plugin:${pluginName}/recommended`
+    const pluginUrl = 'https://github.com/betsys-com/betsys-eslint/tree/main/packages/eslint-plugin-angular-template-spacing'
```

Comment:
Can't we read that info like name/url from package.json? This would be more reliable.